packages/wallets/src/features/cfd/flows/MT5/AddedMT5AccountsList/AddedMT5AccountsList.tsx

```diff
@@ -59,17 +62,50 @@ const AddedMT5AccountsList: React.FC<TProps> = ({ account }) => {
         >
             <div className='wallets-added-mt5__details'>
                 <div className='wallets-added-mt5__details-title'>
-                    <p className='wallets-added-mt5__details-title-text'>{title}</p>
+                    <WalletText size='2xs'>{title}</WalletText>
                     {!activeWallet?.is_virtual && (
                         <div className='wallets-added-mt5__details-title-landing-company'>
-                            <p className='wallets-added-mt5__details-title-landing-company-text'>
-                                {account.landing_company_short}
-                            </p>
+                            <WalletText size='2xs' weight='bold'>
+                                {account.landing_company_short?.toUpperCase()}
+                            </WalletText>
                         </div>
                     )}
                 </div>
-                <p className='wallets-added-mt5__details-balance'>{account.display_balance}</p>
-                <p className='wallets-added-mt5__details-loginid'>{account.display_login}</p>
+                {!(jurisdictionStatus.is_failed || jurisdictionStatus.is_pending) && (
+                    <WalletText size='sm' weight='bold'>
+                        {account.display_balance}
+                    </WalletText>
+                )}
+                <WalletText as='p' color='primary' size='xs' weight='bold'>
+                    {account.display_login}
+                </WalletText>
+                {jurisdictionStatus.is_pending && (
+                    <div className='wallets-added-mt5__details-badge'>
+                        <InlineMessage size='xs' type='warning' variant='outlined'>
+                            <WalletText color='warning' size='2xs' weight='bold'>
+                                Pending verification
+                            </WalletText>
+                        </InlineMessage>
+                    </div>
+                )}
+                {jurisdictionStatus.is_failed && (
+                    <div className='wallets-added-mt5__details-badge'>
+                        <InlineMessage size='xs' type='error' variant='outlined'>
+                            <WalletText color='error' size='2xs' weight='bold'>
+                                Verification failed.{' '}
+                                <a
+                                    onClick={() =>
+                                        show(<VerificationFailedModal />, {
+                                            defaultRootId: 'wallets_modal_root',
+                                        })
+                                    }
+                                >
+                                    Why?
+                                </a>
```

Comment:
SonarCloud's complaining about this, maybe this could fix it

```suggestion
                                <a>
                                    <button
                                        onClick={() =>
                                            show(<VerificationFailedModal />, {
                                                defaultRootId: 'wallets_modal_root',
                                            })
                                        }
                                        style={{ all: 'unset' }}
                                    >
                                        Why?
                                    </button>
                                </a>
```

Reply:
But this is not a button though, its meant to be a clickable link. If I make it to a button I would have to style it to remove the button background and default stylings to make it into a 'link' 🤣

Reply:
`style={{ all: 'unset' }}` does exactly that. Not that I care much about SonarCloud's `onClick` warnings either tho